### DISCUSSION TITLE
how to compare the elements of two lists

### DISCUSSION THREAD

**Discussion by DineshGon** [cypress-io/cypress#14642] [Questions and Help] [answered]
let son1= ["z","g","e","a","b","c","a","e","b"]
       let son2=["a","b","c"]   
       
        cy.get('#elements').children().each(($ele, index, $list) => {
            son1.push($ele.attr('data-term'));
        });

        cy.get('#children').each(($el1, index1, $list1) => {
            son2.push($el1.attr('data-term'));
        })

i would like to write a syntax that compare the array2 values(son2) is preset in array1(son1)

**Comment by bahmutov** [comment]
@fargonmer check out https://glebbahmutov.com/cypress-examples/6.3.0/commands/assertions.html#expect example

**Comment by bahmutov** [comment, marked as answer]
```html
<ol id="first">
  <li>Apples</li>
  <li>Oranges</li>
  <li>Melons</li>
  <li>Grapes</li>
</ol>
<ol id="second">
  <li>Grapes</li>
  <li>Oranges</li>
</ol>
```

```js
const firstList = []
const secondList = []
// let's get the first list of strings
cy.get('#first li').each(($li) => {
  firstList.push($li.text())
})
cy.get('#second li')
  .each(($li) => {
    secondList.push($li.text())
  })
  .then(() => {
    // when this callback runs, both lists will be populated
    expect(firstList).to.include.members(secondList)
  })
```

**Reply by DineshGon** [reply]
Thank you . That helped me to solve my problem

**Reply by kryo-o** [reply]
@bahmutov You are the man!
